types/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dtslint.json",
```

Comment:
This is slightly different than the tslint.json on DefinitelyTyped (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/recurly__recurly-js/tslint.json) because dtslint is picky https://github.com/microsoft/dtslint#typestslintjson